looker-open-source/sdk-codegen
# feat: Add sagas to OAuth Form

Use sagas for OAuth form. This PR includes a suite of tests to ensure sagas are triggered and perform the correct state updates. 

To test:

1. start embed playground
2. ensure verify button doesn't enable with bad url
3. ensure save button doesn't enable with a url that doesn't have versions
4. ensure save button enables with valid URL that returns versions
5. After save config button clicked, ensure login button enables. 
6. After login, be redirected and see 'Hi [NAME], are you ready to embed?'
7. After refresh, still authenticated.
